glib/src/subclass/interface.rs

```diff
@@ -109,6 +109,13 @@ pub trait ObjectInterface: ObjectInterfaceType + Sized + 'static {
     /// in case of interfaces.
     type Prerequisites: PrerequisiteList;
 
+    // rustdoc-stripper-ignore-next
+    /// The C instance struct. This is usually either `std::ffi::c_void` or a newtype wrapper
+    /// around it.
+    ///
+    /// Optional
+    type Instance;
```

Comment:
I was checking and I can't find any way how one could cause problems here by providing some random other type, so I didn't add a new unsafe trait